.pre-commit-config.yaml

```diff
@@ -21,10 +23,10 @@
   - id: check-yaml
   - id: detect-private-key
 
-- repo: git://github.com/FalconSocial/pre-commit-mirrors-pep257
-  sha: master
-  hooks:
-  - id: pep257
+#- repo: git://github.com/FalconSocial/pre-commit-mirrors-pep257
```

Comment:
Agree, but I delayed fixing these errors. I had had this variant before we rebased branch, but this commit was lost.